pos_supplierinfo_barcode/models/product.py

```diff
@@ -0,0 +1,17 @@
+import json
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    # technical field used in POS frontend
+    supplier_barcode_json = fields.Char(
```

Comment:
I recommend renaming this field to `pos_supplier_barcode_json` and to move this comment to the field's `help` attribute.


Not blocking though.